projects/public-search/src/app/document-detail/holdings/holding/holding.component.html

```diff
@@ -0,0 +1,102 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2021 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div id="holding-{{ holding.metadata.pid }}" class="card-header p-2">
+  <div class="row">
+    <div id="holding-location-{{ holding.metadata.pid }}" class="col-sm-4">
+      <a href="#" class="collapse-link mr-2" data-toggle="collapse" aria-expanded="false" (click)="isCollapsed = !isCollapsed">
+        <i [ngClass]="{ 'fa-caret-down': !isCollapsed, 'fa-caret-right': isCollapsed }" class="fa fa-lg" aria-hidden="true"></i>
+      </a>
+      {{ holding.metadata.library.name }}: {{ holding.metadata.location.name }}
+    </div>
+    <div id="holding-category-name-{{ holding.metadata.pid }}" class="col-sm-3">
+      {{ holding.metadata.circulation_category.name }}
+    </div>
+    <div class="col-sm-2">
+      <span [hidden]="itemsCount == 0">
+        {{ itemsCount }} {{ (itemsCount > 1 ? 'items' : 'item') | translate }}
```

Comment:
is this construction allows to extract the strings ?

Reply:
`{{ itemsCount | i18nPlural: { '<=1': 'item', 'other': 'items' } | translate }}`